frappe/website/render.py

```diff
@@ -86,7 +86,9 @@ def render(path=None, http_status_code=None):
 			"Cache-Control": "no-store, no-cache, must-revalidate"
 		})
 
-	return build_response(path, data, http_status_code or 200)
+	headers = {"Link": ",".join(frappe.local.preload)} if hasattr(frappe.local, "preload") else {}
```

Comment:
Shouldn't we preload at the HTML content level itself?

https://developer.mozilla.org/en-US/docs/Web/HTML/Preloading_content

Reply:
What I'm trying to achieve here is HTTP 2 Server Push, It can't function with NGINX without the Link header.